static/game/crosser.js

```diff
@@ -21,8 +21,8 @@ export class Crosser extends Enemy {
     matryoshka = 0
   ) {
     super(pos, vel, acc, matryoshka);
-    this.currentHealth = 1;
-    this.currentHealth = this.maxHealth;
+    this.maxHealth = 25 * (matryoshka + 1);
+    this.gainHealth(this.maxHealth);
```

Comment:
same as previous comment; i have a baseHealth now that takes matryoshka as a function